gravitee-apim-console-webui/package.json

```diff
@@ -20,11 +20,19 @@
     "@asyncapi/web-component": "1.2.26",
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2024-43800   
**Check Name:** serve-static: Improper Sanitization in serve-static   
**Severity:** MEDIUM   
**Fixed Version:** 1.16.0, 2.1.0   
**Reachable Path(s) Found:** No  
**Description:** serve-static serves static files. serve-static passes untrusted user input - even after sanitizing it - to redirect() may execute untrusted code. This issue is patched in serve-static 1.16.0.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2024-43800